Makefile

```
@@ -64,6 +64,8 @@ sec: # Run security static analysis
 build: ## build binaries
 	$(DOCKER_CMD) go build $(GOGCFLAGS) -o "bin/machine-controller-manager" \
                -ldflags "$(LD_FLAGS)" "$(REPO_PATH)/cmd/manager"
+	$(DOCKER_CMD) go build $(GOGCFLAGS) -o "bin/termination-handler" \
+								-ldflags "$(LD_FLAGS)" "$(REPO_PATH)/cmd/termination-handler"
```

Much indent, such space 😜   Possibly we can unindent this a bit
```suggestion
		-ldflags "$(LD_FLAGS)" "$(REPO_PATH)/cmd/termination-handler"
```

---

Not my fault, that's VSCode :D

---

_shakes fist at VSCode_